notifications_utils/url_safe_token.py

```diff
@@ -41,10 +32,4 @@ def check_token(token: str, secret: str | List[str], salt: str = "token", max_ag
     raises BadSignature if the token is invalid
 
     """
-    ser = URLSafeTimedSerializer(secret)
-    try:
-        return ser.loads(token, max_age=max_age_seconds, salt="token")
-    except SignatureExpired:  # SignatureExpired is a subclass of BadSignature so we ensure it is raised first
-        raise
-    except BadSignature:
-        return ser.loads(token, max_age=max_age_seconds, salt=salt)
+    return URLSafeTimedSerializer(secret).loads(token, max_age=max_age_seconds, salt="token")
```

Comment:
Ah interesting, the scripts we worked on yesterday could have an additional salt type then: 'token', which would include time sensitive generated tokens from URLs.

Reply:
right, forgot about this one. We don't really store these in the database or system anywhere though (well, except for in the notification we use to send them to the user) so we probably don't **really** need it. Well, unless we're doing some debugging of the links later, could be useful then. 

I think I'll make a little PR to add it in just in case we want it down the road...